tests/api_tests/nodes/test_views.py

```diff
@@ -2666,7 +2670,8 @@ def test_creates_private_node_pointer_logged_out(self):
     def test_creates_private_node_pointer_logged_in_contributor(self):
         res = self.app.post_json_api(self.private_url, self.private_payload, auth=self.user.auth)
         assert_equal(res.status_code, 201)
-        assert_equal(res.json['data']['attributes']['target_node_id'], self.pointer_project._id)
+        url = res.json['data']['relationships']['target_node']['links']['related']['href']
+        assert_in('/{}nodes/{}/'.format(API_BASE, self.pointer_project._id), url)
```

Comment:
This pattern is copied many times across the tests in this file. To increase maintainability if the URL format ever changes ever change, consider defining all the duplicate code in one place (DRY) and using it as follows.

```python
# Before any tests
node_url_for = lambda n_id: '/{}nodes/{}/'.format(API_BASE, n_id)
...
def test_method(self, ...):
    ...
    expected_url = node_url_for(self.pointer_project._id)
    assert_in(expected_url, url)
```

```